ssh-jump-server/entrypoint.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
-#Set new keys
-rm -rf /etc/ssh/ssh_host_*
-dpkg-reconfigure openssh-server
+if [[ -f /opt/ssh-server-secrets/ssh_host_rsa_key ]]; then
```

Comment:
I think we would ideally allow the admin to choose which algorithm they want to support, rather than assuming and requiring all of these.

Reply:
Updated this to allow any number of keys in the mounted folder with keys. Also updated the key generation script to (optionally) take a list of algorithms to use when generating.